### ISSUE TITLE
Display names of big cities & primary train stations

### ISSUE THREAD

**Issue by thany** [mapsme/omim#6099]
Names of big or medium sized cities oftenly aren't being displayed at any zoom level, making it quite a challenge to figure out what the city is I'm looking at.

A good example of this problem is Aomori, Japan. The only thing that displays by name, is the airport. No matter how far I zoom in or out, the city name is never displayed.

Same goes for the primary train station: even if I zoom in to max level, the train station name is never displayed. I have to tap it to find out what I'm looking at. There's more than enough space to display the name.

Please compare this to Google Maps, which eagerly displays city names, and gradually adds more detail as you zoom in. That's how it should be. Google has the experience on this, so please copypaste their behaviour.

**Comment by brunetton** [comment]
Duplicate of https://github.com/mapsme/omim/issues/12759, and https://github.com/mapsme/omim/issues/13554

**Comment by brunetton** [comment]
Found a simple workaround: use [Organic Maps](https://organicmaps.app/), "real open-source" project

**Comment by thany** [comment]
Yep I found that one as well. I sincerely hope they'll do a better job.
